net/shorewall6/patches/020-init_sh.patch

```diff
@@ -0,0 +1,100 @@
+Index: shorewall6-5.1.4.1/init.openwrt.sh
+===================================================================
+--- /dev/null	1970-01-01 00:00:00.000000000 +0000
++++ shorewall6-5.1.4.1/init.openwrt.sh	2017-06-06 22:52:41.876937057 +0200
+@@ -0,0 +1,95 @@
```

Comment:
Put this in `files/`

Reply:
can be removed because files contains a new init file.
This one was left behind.